src/webgpu/util/math.ts

```diff
@@ -35,7 +35,7 @@ export function clamp(n: number, { min, max }: { min: number; max: number }): nu
 
 /** @returns 0 if |val| is a subnormal f32 number, otherwise returns |val| */
 export function flushSubnormalNumber(val: number): number {
-  return val > kValue.f32.negative.max && val < kValue.f32.positive.min ? 0 : val;
```

Comment:
These subnormal changes where from an earlier revision of the PR, I will move them out of this into a cleanup PR

Reply:
Done